sklearn/feature_selection/base.py

```diff
@@ -120,3 +120,67 @@ def inverse_transform(self, X):
         Xt = np.zeros((X.shape[0], support.size), dtype=X.dtype)
         Xt[:, support] = X
         return Xt
+
+
+def feature_wise_scorer(score_func):
+    """ A wrapper function around score functions.
+
+    Parameters
+    ----------
+    score_func : callable
+        Function taking array(s) X and/or y, and returning a pair of arrays
+        (scores, pvalues) or a single array with scores.
+
+    Returns
+    -------
+    scores : array, shape = [n_features,]
+        Score values returned by the scoring function.
+    p_vals : array, shape = [n_features,]
+        The set of p-values returned by the scoring function.
+
+    Notes
+    -----
+    This wrapper function wraps around scoring functions like `spearmanr`,
+    `pearsonr` etc. from the scipy.stats module and makes it usable for
+    feature selection algorithms like `SelectKBest`.
+
+    Example
+    -------
+    >>> from sklearn.feature_selection import feature_wise_scorer, SelectKBest
+    >>> from scipy.stats import kendalltau, pearsonr, spearmanr
```

Comment:
The `Example` should serve as a quick reference of how to use the function. Just one of these metrics is sufficient.

If you want to demonstrate how this function can apply across different metrics, I suggest updating the [user guide](http://scikit-learn.org/stable/user_guide.html). 

(Don't update the user guide until your code is completely reviewed though. Let's focus on what's important first.)